addons/project/views/project_task_views.xml

```diff
@@ -478,7 +483,7 @@
                                         context="{'default_project_id': project_id}"
                                         column_invisible="not parent.allow_milestones"
                                         invisible="not allow_milestones"/>
-                                    <field name="partner_id" optional="hide" widget="res_partner_many2one" invisible="not project_id"/>
+                                    <field name="partner_id" optional="hide" widget="res_partner_many2one" invisible="not project_id or has_template_ancestor"/>
```

Comment:
I sill think we could completely hide the partner_id field is `has_template_ancestor is True`. What do you think? 🤔